pkg/pillar/types/zedroutertypes.go

```diff
@@ -1817,9 +1817,9 @@ type UnderlayNetworkStatus struct {
 	ACLs int // drop ACLs field from UnderlayNetworkConfig
 	VifInfo
 	BridgeMac       net.HardwareAddr
-	BridgeIPAddr    string // The address for DNS/DHCP service in zedrouter
-	AllocatedIPAddr string // Assigned to domU
-	Assigned        bool   // Set to true once DHCP has assigned it to domU
+	BridgeIPAddr    string   // The address for DNS/DHCP service in zedrouter
+	AllocatedIPAddr []string // Assigned to domU
```

Comment:
I'd be inclined to remame this to AllocatedIPAddrList to make it more clear.

Reply:
split into ip version specific fields